docs/changelog/697.md

```diff
@@ -0,0 +1 @@
+* Added support for planar quads based on triangulation. Quads will be split along the shortest diagonal to maximize the area-to-perimeter ratio. Note that this triangulation will be visible in configured mesh exports.
```

Comment:
```suggestion
* Added support for planar quads based on triangulation, needed, for instance for nearest-projection mapping. Quads will be split along the shortest diagonal to minimize the mapping error. Note that this triangulation will be visible in configured mesh exports.
```

Reply:
Is the use case really worth mentioning here?

Reply:
Yes, that's how users relate to quads. Otherwise we will only get "wait, what do I need quads for?"